### PR TITLE
Note that forcemerges now run in parallel in docs

### PR DESCRIPTION
**Pull request by DaveCTurner**
Relates #69416

### CODE REVIEW

File: docs/reference/indices/forcemerge.asciidoc [hunk 1]
```diff
@@ -68,11 +68,12 @@ You can force merge multiple indices with a single request by targeting:
...
 * All data streams and indices in a cluster
 
-Multi-index operations are executed one shard at a
-time per node. Force merge makes the storage for the shard being merged
-temporarily increase, up to double its size in case `max_num_segments` parameter
-is set to `1`, as all segments need to be rewritten into a new one.
-
+Each targeted shard is force-merged separately using <<modules-threadpool,the
+`force_merge` threadpool>>. This means that the shard-level operations are
+executed in parallel on each node if its `force_merge` threadpool has multiple
```
Comment (ywelsch) [inline docs/reference/indices/forcemerge.asciidoc:73]: I think it's worth mentioning here that the thread-pool defaults to 1 (i.e., this distinction doesn't really matter for most users.)

**Review by ywelsch — COMMENTED** [review]
One small ask, o.w. looking good

File: docs/reference/indices/forcemerge.asciidoc (hunk 1 above)
Comment (DaveCTurner) [reply docs/reference/indices/forcemerge.asciidoc]: Acked, wasn't sure whether to mention that or not. Done in 9afd96ed0a60e754ca34a4db56769c92b8120694.

**Review by ywelsch — APPROVED** [review]
LGTM

**Pull request merged**